service/src/main/resources/config/verily/sdd/original/entity/platform.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "platform",
```

Comment:
[optional] Maybe consider changing the name to "genotyping" or "assay"?

Reply:
Changed to "genotyping" since that's what SD Discover UI has. Reindexed.